release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -434,26 +434,36 @@ The Docs Lead is responsible for updating the Kubernetes API reference, kubectl,
 If you need help in building the reference documentation, reach out on Slack #sig-docs.
 
 ### Generate the reference documentation
+Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)).
 
-Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)). Before running the script, modify `reference.yml` to checkout the Kubernetes future release branch.
-
-![Update kubernetes release branch](pics/reference-yaml.png)
-
-The first step is to famialiarize yourself with the `website/updated-imported-docs/update-imported-docs.py` script
+The first step is to familiarize yourself with the `website/updated-imported-docs/update-imported-docs.py` script
 and the instructions for ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/quickstart/)).
 
-The reference documentation build depends upon a valid release tag.
+The reference documentation build depends upon a valid [release tag](https://github.com/kubernetes/kubernetes/tags).
 When a Kubernetes release candidate version tag such as `v1.19-rc.2` is created, you can consider building
 the reference documentation. Next:
 
 - Create a branch for this work based off of the `dev-[future release]` branch.
+```
+git checkout dev-[future-release]
+git fetch upstream dev-[future-release]
+git rebase upstream dev-[future-release]
+git checkbout -b dev-[future-release]-ref-doc
+```
 - Run the `update-imported-docs.py` script providing the release tag to build the reference documentation.
-- Commit the generated files.
+```
+python website/update-imported-docs/update-imported-docs.py reference.yml [release tag]
+```
 
-![Copy and rename screenshot](pics/copy-rename-dir.png)
+- Commit the generated files.
+```
+git add static/docs/reference/generated/kubernetes-api/v[future-release]/
```

Comment:
There should be many files generated besides the API reference.
BTW, the API reference is currently under development. Expect updates to the process before the 1.20 release.

Reply:
Sometimes, you may need to update configuration files in order to build the kubectl and API references.
This depends from release to release.

Reply:
hmm okay. When I ran it today, it only generated these two files
- static/docs/reference/generated/kubernetes-api/v[future-release]/index.html
- static/docs/reference/generated/kubernetes-api/v[future-release]/js/navData.js

Is there at least a directory I can reference here? I am not a fan of `git add .` 
Is it all at least going to be under the `static/` directory? Then maybe I can change it to `git add static/` instead. What do you think?

Reply:
Did you run the script with one of the 1.20 "rc" tags from kubernetes/kubernetes?
Oh, the release is at v1.20.0-beta.2. You could wait until the first release candidate is available.

Reply:
You only generated two files because these files update a timestamp. The actual content has not changed.
It will become clear when you are using the 1.20 source to build the content.

Reply:
I see! Thanks for the clarification. I'll keep this on hold until next Tuesday. I believe that's when the rc is scheduled. I still think some changes that I've made in the PR are relevant and should be committed but waiting as you suggested until the rc is out makes sense.

Reply:
Hi Karen, git add command in question has been changed to reflect all files.